### ISSUE TITLE
How to remove stroke /lines from radar chart

### ISSUE THREAD

**Issue by jsinhSolanki** [apexcharts/apexcharts.js#284]
Hey, I want to remove lines from radar chart as shown in last img.
here's my jsfiddle
http://jsfiddle.net/jsinh0001/8ekw7hsq/

However if i try to apply transparent stroke it gives like this.
![noradar](https://user-images.githubusercontent.com/31243533/51609698-c8dff800-1f40-11e9-8b77-b62844d0e2ef.PNG)

But I want one border around.

What I want:
![radar](https://user-images.githubusercontent.com/31243533/51609716-db5a3180-1f40-11e9-8f25-ab8c0ffcb6ea.PNG)

**Comment by junedchhipa** [comment]
Although the example you provided is in Vue, I think, it makes sense to move the issue in the core library issues list. 

I am looking more into how single line polygon can be achieved.

**Comment by junedchhipa** [comment]
Added 2 new properties to `plotOptions.radar.polygons`

```javascript
plotOptions: {
  radar: {
    polygons: {
            strokeColors: ['#e8e8e8', 'transparent', 'transparent', 'transparent', 'transparent', 'transparent'],
            connectorColors: 'transparent'
      }
  }
}
```

You will have to wait for the next version 3.2.0 which will be released on Friday

**Comment by junedchhipa** [comment]
Released 3.2.0
